### ISSUE TITLE
Key action icons

### ISSUE THREAD

**Issue by Nixill** [Shinmera/promptfont#24]
It would be nice to have icons for performing specific actions with a controller button.

This is Steam Input's list of which actions on a button or stick can activate a control, which I figured would be a good starting point:
1. Regular press
2. Double press
3. Long press
4. Start of press
5. Release of press
6. Analog range
7. Soft press
8. Button chord

Presumably, 1 can just be expressed as the lack of an action icon - but I also figured that having something that explicitly says "just tap, don't do anything else" could be neat, so I made it.

2 through 5 are also present below.

6 and 7 could possibly see a use on joysticks and triggers, but I didn't feel a need to make glyphs for those.

And 8 can just be expressed by a + between two buttons. You don't need anything special for it.

With that said, I've created glyphs for 1-5 as seen below:
![Glyphs for single tap, double tap, long tap, start tap, and end tap](https://github.com/Shinmera/promptfont/assets/1627071/a6bf1835-7a33-4bdf-9736-86aea4905395)

Notably, single and long tap aren't quite centered in the document, but I went for consistency of the button's location compared to the other glyphs in this set for those two.

**Comment by Shinmera** [comment]
Those look great!

The only one I'm a bit concerned about is the third one, It might not be clear enough especially when put compared to the taps. Maybe something like this could be clearer?

![2024 05 20 22:25:40](https://github.com/Shinmera/promptfont/assets/4833204/69427922-697b-43a3-a266-68e0a1db4d7f)

I'd happily merge a PR if you can submit one according to the [contribution guidelines](https://github.com/Shinmera/promptfont/blob/master/CONTRIBUTING.md) in any case.

**Comment by StevenH237** [comment]
Updated! I agree, the "..." isn't very clear. While I was at it I decided to switch the plain oval for the empty keycap already available in the font, modified to better fit the dimensions.

They're also now sorted below by filename, which I would suggest as `action/press-end.svg`, `action/press-long.svg`, `action/press-start.svg`, `action/tap-double.svg`, and `action/tap-single.svg`. If you don't want to create a new folder, I suppose they fit as icons (so `icon/action-` instead of `action/`).

![image](https://github.com/Shinmera/promptfont/assets/1627071/1531d1f0-f20c-49d1-af3d-dd3a83b969a1)
